core/store/src/main/java/uk/gov/gchq/gaffer/store/library/FileGraphLibrary.java

```diff
@@ -95,7 +95,7 @@ public Pair<String, String> getIds(final String graphId) {
 
     @Override
     protected void _addIds(final String graphId, final Pair<String, String> schemaAndPropsIds) throws OverwritingException {
-        String schemaAndPropsIdsString = new String(schemaAndPropsIds.getFirst() + "," + schemaAndPropsIds.getSecond());
+        String schemaAndPropsIdsString = schemaAndPropsIds.getFirst() + "," + schemaAndPropsIds.getSecond();
```

Comment:
`String.join` may be neater

Reply:
That could be done, though it's a longer line of code (even with static import) and from what I've read there's no performance benefit.